packages/workers/src/types/jobs.ts

```
@@ -0,0 +1,292 @@
+  agentId?: string;
+  userId?: string;
+  createdAfter?: Date;
+  createdBefore?: Date;
+  limit?: number;
+  offset?: number;
+}
+
+/**
+ * Job metadata filters (alias for backward compatibility)
+ */
+export interface JobMetadataFilters extends JobQueryFilters {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)